doc/man/cgexec.1

```diff
@@ -49,6 +49,11 @@ kernel values.
 .B -h, --help
 Display this help and exit.
 
+.TP
+.B -r
+Replaces systemd scope's idle process with the task \fBcommand\fR.
```

Comment:
Same comment as earlier.  We need to use consistent nomenclature for the idle thread.

Now what that should be?... Well, I'm not sure yet :).